styleguide-app/Examples/Icon.elm

```diff
@@ -195,7 +195,6 @@ viewIcon { alt, background, icon } =
             , title alt
             ]
             [ Icon.icon { alt = alt, icon = icon assets }
```

Comment:
oh, there was a change her. I'll go back...


Meta:
There was a guide on how to/when to create a new version right? I think we should include a guideline in that to create a commit with an exact copy of the current version and make the changes on top of that commit. This would make reviews a lot simpler.